app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QuotesController < ApplicationController
+
+  def index
+    @searched_string = params[:query]
+    @quotes_total = QuoteFacade.get_total(@searched_string)
+    @quotes = QuoteFacade.get_quotes_info(@searched_string)
+  end
```

Comment:
Your `get_quotes_info` method returns the total count. Even in your facade, your `get_total` method calls to the `get_quotes_info` method. So, on both line 5 and 6, you're either directly or indirectly calling `get_quotes_info`. Since you can get both total quote count AND quote info from one api call, you can do something like this: 

```ruby
class QuotesController < ApplicationController

  def index
    @searched_string = params[:query]
    @quotes_total, @quotes  = QuoteFacade.get_total(@searched_string)
  end

end
```

```ruby
class QuoteFacade
  def self.get_quotes_info(searched_string)
    json = QuoteService.get_quotes_info(searched_string)
    quotes = json[:results].map do |quote_data|
      Quote.new(quote_data)
    end
  total_count, quotes = [json[:totalCount], quotes]
  end 
end
```